digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -361,7 +386,12 @@ ObjectNode fetchLogEvents(final EcsClient client,
         }
         else {
             for (final OutputLogEvent logEvent : logEvents) {
-                log(logEvent.getMessage() + "\n", clog);
+                String log = logEvent.getMessage();
+                if (log.contains(ECS_TASK_FINISHED)) {
+                    nextExecutorStatus.set("logging_finished", JsonNodeFactory.instance.textNode("true"));
```

Comment:
I recommend to use `logging_finished_at` and set timestamp instead of `true` because we can know when waiting logging has finished without much overhead. It helps us to investigate problems if some happened.

Reply:
Sorry I forgot to fix, and I fixed to use time here.
https://github.com/treasure-data/digdag/pull/1316/commits/b45c5fb60f7480a6ad6c21f9d71d90ed663ae23b